src/components/filter/MessagesFilterPanel.tsx

```diff
@@ -293,7 +293,11 @@ const MessagesFilterPanel = () => {
 				renderFooter={renderFooter}
 			/>
 			<MessageReplayModal />
-			<MessageFilterWarning />
+			{messagesStore.checkingAttachedMessages ? (
```

Comment:
There is already loader in this component. 2 loaders would be to much i think

FilterPanel accepts isLoading prop, we can pass it there